### ISSUE TITLE
Exception calling "Add" on filename already recorded in dictionary @ public.ps1:264 char:18

### ISSUE THREAD

**Issue by mavaddat** [jdhitsolutions/PSReleaseTools#16]
### PSVersion Table
| Name                      	| Value                   	|
|---------------------------	|-------------------------	|
| PSVersion                 	| 5.1.19555.1001          	|
| PSEdition                 	| Desktop                 	|
| PSCompatibleVersions      	| {1.0, 2.0, 3.0, 4.0...} 	|
| BuildVersion              	| 10.0.19555.1001         	|
| CLRVersion                	| 4.0.30319.42000         	|
| WSManStackVersion         	| 3.0                     	|
| PSRemotingProtocolVersion 	| 2.3                     	|
| SerializationVersion      	| 1.1.0.1                 	|

PSCompatibleVersions:

| Major 	| Minor 	| Build 	| Revision 	|
|-------	|-------	|-------	|----------	|
| 1     	| 0     	| -1    	| -1       	|
| 2     	| 0     	| -1    	| -1       	|
| 3     	| 0     	| -1    	| -1       	|
| 4     	| 0     	| -1    	| -1       	|
| 5     	| 0     	| -1    	| -1       	|
| 5     	| 1     	| 19555 	| 1001     	|

### What is the syntax or expression you are using?
`Install-PSPreview -mode Passive`

### What happens?

![image](https://user-images.githubusercontent.com/5055400/73586625-d9d11c80-4464-11ea-8d26-b7d496a6b9e2.png)
```powershell
 Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process
```
```powershell
if(Get-InstalledModule -Name PSReleaseTools -ErrorAction Ignore) {Import-Module PSReleaseTools; Install-PSPreview -mode Passive} else {$false}
```
```
ForEach-Object : Exception calling "Add" with "2" argument(s): "Item has already been added. Key in dictionary:
'powershell-7.0.0-rc.2-linux-x64.tar.gz'  Key being added: 'powershell-7.0.0-rc.2-linux-x64.tar.gz'"
At C:\Program Files\WindowsPowerShell\Modules\PSReleaseTools\1.4.1\functions\public.ps1:264 char:18
```
```powershell
+             $r | ForEach-Object -Begin {
+                  ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [ForEach-Object], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentException,Microsoft.PowerShell.Commands.ForEachObjectCommand
```

### What do you think should happen?
`public.ps1` should only **try** to add the filename to the dictionary object without assuming that it will be unique.

**Comment by jdhitsolutions** [comment]
The odd thing is that it is trying to add the linux package. Something definitely not right.

**Comment by jdhitsolutions** [comment]
The bug is actually in `Get-PSReleaseAsset` and I wonder if MS changed their asset structure.

**Comment by jdhitsolutions** [comment]
The release note includes a reference to a replaced file:

```text

**Note:** This file was replaced because it was missing symbolic links.

- powershell-7.0.0-rc.2-linux-x64.tar.gz
  - F365C5DE1D29975B5CB759C16BBF636326311C848410AA397491BE247DE7B732
```

`Get-PSReleaseAsset1 is detecting this as a file to download.

**Comment by jdhitsolutions** [comment]
Fixed in v1.5.0 which has been pushed to the PowerShell Gallery. @mavaddat Update and try again.

**Comment by mavaddat** [comment]
Works correctly now. Thanks!
